datafusion/core/tests/sqllogictests/test_files/timestamps.slt

```diff
@@ -76,6 +76,18 @@ SELECT DATE_BIN(INTERVAL '15 minutes', TIMESTAMP '2022-08-03 14:38:50Z', TIMESTA
 ----
 2022-08-03T14:30:00
 
+# Can coerce string interval arguments
+query T
+SELECT DATE_BIN('15 minutes', TIMESTAMP '2022-08-03 14:38:50Z', TIMESTAMP '1970-01-01T00:00:00Z') AS res
```

Comment:
small nit: if AS res is needed?

Reply:
I think you are right -- `res` is not needed. I will remove them

Reply:
In 7a7fd4b92